recipes/openimageio/all/CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+cmake_minimum_required(VERSION 3.12)
+project(cmake_wrapper)
+
+set(CONAN_BASIC_SETUP_ARGS TARGETS)
+if(APPLE)
+    # When building shared libraries,
+    # the binaries in the build tree need RPATHS.
+    list(APPEND CONAN_BASIC_SETUP_ARGS KEEP_RPATHS)
+    set(CMAKE_SKIP_INSTALL_RPATH 1)
+endif()
+
+include("${CMAKE_CURRENT_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup(${CONAN_BASIC_SETUP_ARGS})
+
+# The following code disables CMake's find_package command
+# by replacing it with a no-op stub and directly sets
+# all necessary variables to values provided by respective
+# Conan packages.
+
+function(find_package)
+    message(STATUS "[STUB] find_package(${ARGV})")
+endfunction()
+
+set(BOOST_CUSTOM YES)
```

Comment:
I see many dependencies declared here, but the project provides a cmake file. Maybe you should add cmake_find_package generator.

Reply:
I switched to `cmake_find_package`. In order to make this work, I had to create a bigger patch, since some of OIIO's custom find modules don't use standard name conventions. Is that ok?